boards/arm/nrf9160_innblue22/nrf9160_innblue22_common.dts

```diff
@@ -0,0 +1,219 @@
+/*
+ * Copyright (c) 2020 InnBlue
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/ {
+	model = "innblue v22 Dev Kit";
+	compatible = "innblue,innblue22",
+				"nordic,nrf9160-sica", "nordic,nrf9160";
+
+	chosen {
+		zephyr,console = &uart0;
+		zephyr,shell-uart = &uart0;
+		zephyr,uart-mcumgr = &uart0;
+	};
+
+	leds {
+		compatible = "gpio-leds";
+		red_led: led_1 {
+			gpios = <&gpio0 7 0>;
+			label = "RGB red channel";
+		};
+		green_led: led_2 {
+			gpios = <&gpio0 6 0>;
+			label = "RGB green channel";
+		};
+		blue_led: led_3 {
+			gpios = <&gpio0 5 0>;
+			label = "RGB blue channel";
+		};
+
+		mode_led: led_4 {
+			gpios = <&gpio0 4 0>;
+			label = "mode red channel";
+		};
+	};
+
+	buttons {
+		compatible = "gpio-keys";
+
+		button0: button_0 {
+			gpios = <&gpio0 31 GPIO_PUD_PULL_UP>;
+			label = "Button 0";
+		};
+	};
+
+	/* These aliases are provided for compatibility with samples */
+	aliases {
+		led0 = &red_led;
+		led1 = &green_led;
+		led2 = &blue_led;
+		led3 = &mode_led;
+		sw0  = &button0;
+		rgb-pwm  = &pwm0;
+		mode-pwm = &pwm1;
+	};
+};
+
+&gpiote {
+	status = "okay";
+};
+
+&gpio0 {
+	status = "okay";
+};
+
+&uart0 {
+	status = "okay";
+	current-speed = <115200>;
+	tx-pin = <29>;
+	rx-pin = <30>;
+
+};
+
+&uart1 {
+	status = "okay";
+	current-speed = <115200>;
+	tx-pin = <14>;
+	rx-pin = <15>;
+};
+
+&uart2 {
+	current-speed = <115200>;
+	tx-pin = <18>;
+	rx-pin = <19>;
+};
+
+
+&i2c2 {
+	compatible = "nordic,nrf-twim";
+	status = "okay";
+	sda-pin = <25>;
+	scl-pin = <26>;
+
+	clock-frequency = <I2C_BITRATE_FAST>;
+
+	bq27421@55 {
+		compatible = "ti,bq274xx";
+		label = "BQ274XX";
+		reg = <0x55>;
+	};
+
+	lis2dh12-accel@19 {
+		compatible = "st,lis2dh";
+		reg = <0x19>;
+		irq-gpios = <&gpio0 0 0>, <&gpio0 1 0>;
```

Comment:
Use `GPIO_ACTIVE_HIGH` as flags (the ending zeros) for both these GPIOs, for consistency with other uses the sensor in the Zephyr tree.